lib/client/src/account_fetcher.rs

```diff
@@ -29,6 +29,8 @@ pub trait AccountFetcher: Sync + Send {
         program: &Pubkey,
         discriminator: [u8; 8],
     ) -> anyhow::Result<Vec<(Pubkey, AccountSharedData)>>;
+
+    fn rpc(&self) -> &RpcClientAsync;
```

Comment:
My intuiton is that this abstraction shouldn't require being tied to the particular RpcClientAsync. Looks like the only reason for it is that fetch_multiple isn't a primitive on the AccountFetcher trait - add that instead.